llvm/test/Instrumentation/AddressSanitizer/remove-memory-effects.ll

```diff
@@ -0,0 +1,23 @@
+; Remove possible memory effects from functions that are invalidated by
+; AddressSanitizer instrumentation.
+
+; RUN: opt < %s -passes=asan -asan-use-after-scope -S | FileCheck %s
```

Comment:
It would be better if this cl::opt was moved to be parsed from the pass pipeline parameters